### PR TITLE
Added some useful features

### PR DESCRIPTION
**Pull request by shaoziyang**
Added some useful features, such as:

- page view count
- site visit count
- recent pages
- page and image files count
- random pages
- wallpaper
- image library
- page effects scripts

It is very easy to configure, DIY various styles.

### CODE REVIEW

**Comment by Zavy86** [comment]
oh wow! *_*

**Comment by vCoDMods** [comment]
> oh wow! 🤩

Will you merge this to master?

**Review by Zavy86 — CHANGES_REQUESTED** [review]

File: common.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+function console_log($data){
+    if (is_array($data) || is_object($data))
+        echo("<script>console.log(".json_encode($data).");</script>");
+    else
+        echo("<script>console.log('".$data."');</script>");
+}
+
+// random int number per day
+function randint_day(){
```
[the remark this answers is not in the document]
Comment (shaoziyang) [reply common.php:11]: generate daily random numbers for daily quotes

File: scripts/opacity_10.png
[the remark this answers is not in the document]
Comment (shaoziyang) [reply scripts/opacity_10.png:1]: Used to set the transparency of the background image by overlaying the image.

**Comment by shaoziyang** [comment]
Please ignore this pull request, I will submit a new one.

**Pull request closed without merging**